x-pack/plugins/security/server/routes/views/login.ts

```diff
@@ -62,7 +62,8 @@ export function defineLoginRoutes({
       security: {
         authz: {
           enabled: false,
-          reason: 'This route is opted out from authorization',
+          reason:
+            'This route is opted out from authorization because it only provides non-sensative information about authentication provider configuration',
```

Comment:
Not really sure about this one.

Reply:
This route provides the necessary information to render the login selector UI. In other words, this is what gives users the choice of how they wish to authenticate. Since they haven't yet authenticated, we can't require authorization.